scripts/ci/setup-ipfs.sh

```diff
@@ -46,7 +46,7 @@ echo "::group::Preconnect to cluster peers"
         --basic-auth "${CLUSTER_USER}:${CLUSTER_PASSWORD}" \
         peers ls > cluster-peers-ls
     for maddr in $(jq -r '.[].ipfs.addresses[]?' cluster-peers-ls); do
-        ipfs swarm connect "$maddr" || continue
```

Comment:
`ipfs swarm peering add` seems to wait a few seconds before doing a dial. While this seems fine it might be helpful to do an `ipfs swarm connect` after the `ipfs swarm peering add` just so `ipfs swarm peers` shows more peers in it.